synthesizer/tests/tests/vm/execute_and_finalize/mint_and_split.aleo

```diff
@@ -17,7 +17,7 @@ cases:
     private_key: APrivateKey1zkpJhviKDvvm7yu7SZuhSudVR7zjCRG2HznuAHwuGYc1xqN
   - program: mint_and_split.aleo
     function: split
-    inputs: [ "{ owner: aleo1j2hfs6yru47h2nvsjdefwtw6nwaj0y4zcl02juyy29txm7nt6y9qln7uhp.private, microcredits: 100u64.private, _nonce: 4745749250363947745807119124444606639539806979883854068889616498365610280135group.public, _version: 1u8.public }", 50u64]
+    inputs: [ "{ owner: aleo1j2hfs6yru47h2nvsjdefwtw6nwaj0y4zcl02juyy29txm7nt6y9qln7uhp.private, microcredits: 100u64.private, _nonce: 8328661033496413191179695268319516441428135407695959353783510885778770196235group.public, _version: 1u8.public }", 50u64]
```

Comment:
So I guess the previous value became invalid because some internal RNG state of the test changed? Which will in all likelihood happen again soon.

As such, this PR has no value, it'll be broken again soon. We should probably move the testing of size and fee estimation with inclusion and translation to `test_v12.rs`.